src/main/java/sirius/db/mixing/properties/AmountProperty.java

```diff
@@ -111,14 +114,21 @@ protected Object transformValueFromImport(Value value) {
 
     @Override
     protected Object transformToJDBC(Object object) {
-        return object == null || ((Amount) object).isEmpty() ? null : ((Amount) object).getAmount();
+        Amount amount = (Amount) object;
+        if (amount == null || amount.isEmpty()) {
+            return null;
+        }
+        NumberFormat format = getAnnotation(Numeric.class).map(numeric -> {
+            return new NumberFormat(numeric.scale(), RoundingMode.HALF_UP, NLS.getMachineFormatSymbols(), false, null);
+        }).orElse(NumberFormat.MACHINE_THREE_DECIMAL_PLACES);
+        return amount.toString(format);
```

Comment:
Are you sure we want to do this here, as this also influences the transformation of actual values, which were fine before, i assume ?

Maybe a better way would be to overwrite getColumnDefaulValue in AmountProperty

Reply:
In general, this shouldn't be a problem - the values cannot be stored with higher precision in the DB. But as there might be some edge cases, I agree.